backend/app/alembic/versions/d64a4e1708bd_add_column_theme_id_to_inquiry_model.py

```diff
@@ -0,0 +1,29 @@
+"""Add column theme_id to Inquiry model
+
+Revision ID: d64a4e1708bd
+Revises: 828527f72c54
+Create Date: 2024-11-26 05:15:15.016183
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = 'd64a4e1708bd'
+down_revision = '828527f72c54'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('inquiry', sa.Column('theme_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(None, 'inquiry', 'theme', ['theme_id'], ['id'], ondelete='CASCADE')
```

Comment:
stupid question -- does this mean if the last inquiry referencing a particular theme gets deleted, the theme gets deleted too? do we want that?

Reply:
My understanding is inquiries would be deleted if the related theme is deleted, I don't think we want this behavior either way thought now that you mention it. I will fix this.